src/Cassandra.IntegrationTests/Cassandra.IntegrationTests.csproj

```diff
@@ -102,20 +102,13 @@
     <PackageReference Include="NUnit" Version="3.14.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="NunitXml.TestLogger" Version="2.1.41" />
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-    <PackageReference Include="System.Runtime.Serialization.Json" Version="4.0.2" />
-    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
-    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
Why this change

Reply:
I think it's built in .NET Standard 2.1 so it was generating some warning about it.

Reply:
https://github.com/datastax/csharp-driver/pull/617#discussion_r2046999702

Reply:
Could not reproduce the warning anymore 🤔 Added the code back.